test/unittests/state_transition_test.cpp

```diff
@@ -0,0 +1,28 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "../utils/bytecode.hpp"
+#include "state_transition.hpp"
+
+using namespace evmc::literals;
+using namespace evmone::test;
+
+TEST_F(state_transition, eof_invalid_initcode)
+{
+    static constexpr auto create_address = 0x864bbda5c698ac34b47a9ea3bd4228802cc5ce3b_address;
+
+    pre.get(*tx.to).code = eof1_bytecode(create() + push(1) + OP_SSTORE + OP_STOP, 3);
```

Comment:
the ongoing discussion is wether to have an individual tool that will verify validity of EOF 
cause strictly there are EOF codes that are valid, but represent an invalid evm code sequence. so it can't be tested through state transition

Reply:
what are you talking about, what's invalid evm code sequence?

Reply:
In eth rnd execution layer testing channel.